services/security/keyvault/keyvault.go

```diff
@@ -200,6 +202,39 @@ type Key struct {
 	*KeyProperties `json:"properties,omitempty"`
 }
 
+// KeyWrappingAlgorithm enumerates the values for private key wrapping info for key import export operations.
```

Comment:
https://github.com/Azure/azure-sdk-for-go/blob/main/services/keyvault/v7.0/keyvault/models.go

am looking at azure sdk for reference. Would prefer to have same APIs